backend/apps/sandbox/consumers.py

```
@@ -283,37 +283,89 @@ async def connect(self):
+            def get_and_verify_session():
+                try:
+                    session = CollabSession.objects.filter(
+                        id=self.room_id, is_active=True
+                    ).first()
+                    if session:
+                        is_owner = session.project and session.project.user == user
+                        is_allowed = session.allowed_users.filter(id=user.id).exists()
+                        if is_owner or is_allowed:
+                            return session
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)